module-2/backend/week_two.md

```diff
@@ -1,35 +1,64 @@
 ## Week Two - Module 2 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week (which is a TON - YOU are a web developer!!!). 
+Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week (which is a TON - YOU are a web developer!!!).
 
 Note: When you're done, submit a PR.
 
 1. At a high level, what is ActiveRecord? What does it do/allow you to do?
-2. Assume you have the following model:
+ActiveRecord takes rows of data from a table and turns each row into an instance of a class, where the whole table is mapped to the class,and each row is an instance of the class. Each column is turned into a method that can be called on the instance to return the value of the data stored in the associated row/column. It allows you to access and use the data via classes, and lets you link different tables together in order to access info from different tables that are related to each other.
 
+2. Assume you have the following model:
 ```ruby
 class Team << ActiveRecord::Base
 end
 ```
-
 What are some methods you can call on `Team`? If these methods aren't defined in the class, how do you have access to them?
+You could call .all, .count, .find, .find_by, .where, which are all methods defined by ActiveRecord.
 
 3. Assume that in your database, a team has the following attributes: "id", "name", owner_id". How would you find the name of a team with an id of 4? Assuming your class only included the code from question 2, how could you find the owner of the same team?
+Team.find(4).name
+Team.find(4).owner
```

Comment:
Without the `belongs_to` you would need to be a little creative to find the owner.

```
team = Team.find(4)
Owner.find(team.owner_id)
```